src/main/java/edu/rosehulman/quota/Application.java

```diff
@@ -2,13 +2,16 @@
 
 import static spark.Spark.get;
 import static spark.Spark.port;
+import static spark.Spark.post;
 
 public class Application {
   public static void main(String[] args) {
     port(8080);
 
     get("/health", (request, response) -> "OK");
 
+    post(Paths.ADD_USER, new AddUserController()); // Consume an AddUser call from SDK/partner
```

Comment:
After many sleepless nights, I regret my idea of using a Paths static class and propose we just embed the paths string right here.

Reply:
Why do you want to get rid of the Paths class?

Reply:
The Paths class is cleaner, but seeing the path string right here might be better for usability purposes and is one less level of indirection. We can do it either way, I was just saying that if it were put it up to a vote, I would vote to put the string right in this file.

Reply:
Ok that's fine.  This way we can also see if the path is post, get, or put